### PR TITLE
Allow any active user to be selected as a teammate

### LINKED ISSUE #335: Mappers may select a teammate who assisted them with mapping
Refers to Task 28 in the scope - Support Team Mapping. A team can be no more than two people so the user entering the data can only note that one other person helped them with mapping.

### PR DESCRIPTION
**Pull request by kdeloach**
Fixes #335

### CODE REVIEW

File: src/nyc_trees/apps/survey/helpers.py [hunk 1]
```diff
@@ -10,16 +10,16 @@
...
-    attendees = User.objects.exclude(id=current_user.id)\
-                            .filter(eventregistration__event=event,
-                                    eventregistration__did_attend=True)\
+    attendees = User.objects.exclude(id=current_user.id) \
+                            .filter(is_active=True) \
                             .order_by('username')
     return _teammates_context(attendees)
 
 
 def teammates_for_individual_mapping(current_user):
```
Comment (kdeloach) [inline src/nyc_trees/apps/survey/helpers.py]: Could probably combine these functions...

Comment (maurizi) [inline src/nyc_trees/apps/survey/helpers.py]: Yeah, I think that would be a good idea.

Comment (kdeloach) [inline src/nyc_trees/apps/survey/helpers.py]: Updated.

**Comment by maurizi** [comment]
+1, works great!

**Pull request merged**